deploy/registry/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry:2
+FROM registry:2.6.2
 RUN apk add --no-cache --update curl
```

Comment:
A solution that would last the tests of time would be the following:

```
RUN apk add --no-cache --update curl apache2-utils
```

Instead of pinning to an earlier version.